comfy/model_management.py

```diff
@@ -21,7 +22,12 @@
 
 try:
     import torch
-    total_vram = torch.cuda.mem_get_info(torch.cuda.current_device())[1] / (1024 * 1024)
+    import intel_extension_for_pytorch as ipex
```

Comment:
This is going to break auto lowvram detection on CUDA.

Reply:
> This is going to break auto lowvram detection on CUDA.

I have placed the total_vram for CUDA under:

```py
if torch.xpu.is_available():
    xpu_available = True
    total_vram = torch.xpu.get_device_properties(torch.xpu.current_device()).total_memory / (1024 * 1024)
else:
    total_vram = torch.cuda.mem_get_info(torch.cuda.current_device())[1] / (1024 * 1024)
```

Reply:
> I have placed the total_vram for CUDA under:

Well, it looks like this else will not work when there is no IPXE...

Reply:
Should be fixed. https://github.com/comfyanonymous/ComfyUI/pull/409/commits/3e2608e12b312fd5d2396d4146d992cd4f8b9ab4